### ISSUE TITLE
Add timing info to "sails lift --verbose"

### ISSUE THREAD

**Issue by kevinburke** [balderdashy/sails#2495]
It would be useful to know how much time each task takes. Either a running output like

```
verbose (0.01): Setting Node environment...
verbose (0.03): moduleloader hook loaded successfully
verbose (0.14): Loading app config..
```

or append timing data to each line

```
verbose: Setting node environment... +1ms
verbose: moduleloader hook loaded successfully. +2ms
```

**Comment by loicsaintroch** [comment]
I agree. We already have something like this for `locales` but it could be great to have this on everything.

**Comment by mikermcneil** [comment]
Thanks for posting, @kevinburke.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
